fix: copy button of code block menu doesn't work

qinluhe opened the pull request:
closes https://github.com/toeverything/blocksuite/issues/1145

lawvs commented on packages/editor/src/managers/clipboard/copy-cut-manager.ts:
```diff
@@ -216,6 +214,8 @@ export class CopyCutManager {
     } finally {
       tempElem.removeEventListener('copy', listener);
       document.body.removeChild(tempElem);
+
+      const curRange = getCurrentRange();
```
Please revert this change, it will break copy action in the format bar (selection lost)


https://user-images.githubusercontent.com/18554747/218300412-d4dea242-c77d-4433-97ed-b3a6f93a50b8.mov

thorseraq commented:
We need further investigation for #1128, key point is `_getClipItems()` during copy, and `_clipboardData2Blocks()` during paste. For example, now hit copy button and paste it at a line end of paragraph, it won't append a new code block.

But it fixes #1145, could you add a unit test in `tests/code.spec.ts`?

qinluhe commented:
> We need further investigation for #1128, key point is `_getClipItems()` during copy, and `_clipboardData2Blocks()` during paste. For example, now hit copy button of code block and paste it at a line end of paragraph, it won't append a new code block.
> 
> But It fixes #1145, could you add a unit test in `tests/code.spec.ts`?

1. There are two different reasons for the problem that clicking the copy button does not meet expectations, one of which is that the copy event is not passed to handleCopy correctly, this PR fixes the problem for this reason

2. 👌🏻

thorseraq commented:
@qinluhe I have added the unit test, thanks for contributing! 

we have added the CLA process in BlockSuite. Would you mind signing it by editing on https://github.com/toeverything/blocksuite/edit/master/.github/CLA.md ? Thanks!

qinluhe commented:
> @qinluhe I've added the unit test, thanks for contributing!
> 
> we have added the CLA process in BlockSuite. Would you mind signing it by editing on https://github.com/toeverything/blocksuite/edit/master/.github/CLA.md ? Thanks!
👌🏻
